src/TabContext.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const TabContext = React.createContext(
+  null /* {
```

Comment:
?

Reply:
i was signaling to myself waht this type will be, but leaving it null to test presence. Good example of where types would be nice.